src/api/transactions/transaction.spec.ts

```diff
@@ -24,6 +29,27 @@ describe("Wiremessages", () => {
0000000000000000000000000000000000000000000000000004033f7ef5bcef66ea157f22ece0b7941897c67fa4bb2fd8d1af25224e9a2d24fcef2c182b9a127700c4025f9ec0c98cc35b94d51c88aa553cfd096b97f7586e2be00000000000000000000000090a178bdf2018e4cb104b28f6276cdfa0e32f1fd0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006038424b50e731a15664fc82bb4d7a9288a20effb7092a477a30f3eca90122c7dc5d4350a213f20b5c521a7877e09ff95d7eee15c975c8280b952a2fa57ad8bac584f9d82636fe8a455a57af682c1a88fedf173ae0c17af601496d643b09f590c9000000000000000000000000ab5e120e45085aaaaac8f81d21087c8c6cec81ce00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000020617683715581a365caf4354ecb69b248fbde705ccdbd33b6ef3048344407ea920000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000e00000000000000000000000000000000000000000000000000000000000000160000000000000000000000000ac15eca362aeae4876e76fbd5648130e8f123f290000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000002066cccdc1cb7da5c70669401f4b617b26a69ae3df1598d4a781f9927b6b42a5ce000000000000000000000000d26267f82969fba5d6be0986f5a505f1604107b5000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000207932032e2e9e3ca0183fa006dcf696e2263b04ab84da3c907afb98c50b3ffa0d000000000000000000000000d740ae29e3d69e2f42899e413381e1553840f72a00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000020133cfb4f6aafb36324186e6c6da8d56cf1812d048e5a39f3db714c2a195b042d000000000000000000000000000000000000000000000000000000000000022000000000000000000000000000000000000000000000000000000000000000800000000000000000000000006e2fd8a6fb6caf3612967aadc749763e9657e9d100000000000000000000000049ed509ee584a17987156e276f180b234e79292c00000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000114572647374616c6c5472616465466565730000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000035b7ef7803eb1a954517ee58aa9ea5964afd44ce000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000203aeb4f544769e0a8dd5ab3d2a8569336cadeb54db0af60cf79c5c13f8b1c1b86000000000000000000000000ccf0fbe58471cb5e6e4bf25fddb8f8709b24f1fc00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000020118d5931da69e0069fcb94ef71e248bb22379080bd8983dda622c7c9a59416df");
```

Comment:
Where did you derive the golden encodings from? Please add a comment to document how they were derived in the first place.